content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md

```diff
@@ -0,0 +1,223 @@
+I vividly remember the last minute PRs from last release because I was trying to cut the releases, as part of the branch management team. Thankfully, that was not the case this release. We have had other challenges, of course. 
+
+**Can you tell me some of those challenges?**<br>
+I think improvement on documentation is always a big part. The KEP process can be very daunting to new contributors. How do you get people to review your KEPs? How do you opt in? All that stuff. We're improving documentations for that. 
+
+**As someone who has been through a lot of releases, I've been feeling, like you've said, that the last minute nature has slowed down a little. The process is perhaps improving. Do you see that, or do you think there's still a long way to go for the leads to improve it?**<br>
+I think we've come very far. When I started in 1.22, we were using spreadsheets to track a hundred enhancements. It was a monster; I was terrified to touch it. Now, we're on GitHub boards. As a result of that, we are actually merging the bug triage and CI Signal team in 1.29. 
+
+**What's the impact of that?**<br>
+The bug triage team is now using the GitHub board to track issues, which is much more efficient. We are able to merge the two teams together.
+
+**I have heard a rumor that GitHub boards are powered by spreadsheets underneath.**<br>
+Honestly, even if that's true, the fact that it's on the same platform and it has better version control is just magical. 
+
+**At this time, the next release lead has not yet been announced, but tradition dictates that you write down your feelings, best wishes and instructions to them in an envelope, which you'll leave in their desk drawer. What are you going to put inside that envelope?**<br>
+Our 1.28 release lead is fantastic and they're so capable of handling the release—
+
+**That's you, isn't it?**<br>
+1.29? [LAUGHS] No, I'm too tired. I need to catch up on my sleep. My advice for them? It's going to be okay. It's all going to be okay. I was going to echo Leo's and Cici's words, to overcommunicate, but I think that has been said enough times already. 
+
+**You've communicated enough. Stop! No more communication!**<br>
+Yeah, no more communication. [LAUGHS] It's going to be okay. And honestly, shout out to my emeritus advisor, Leo, for reminding me that. Sometimes there are a lot of fires and it can be overwhelming, but it will be okay. 
+
+**As we've alluded to a little bit throughout our conversation, there are a lot of people in the Kubernetes community who, for want of a better term, have had "a lot of experience" at running these systems. Then there are, of course, a lot of people who are just at the beginning of their careers; like yourself, at university. How do you see the difference between how those groups interact? Is there one team throughout, or what do you think that each can learn from the other?**<br>
```

Comment:
```suggestion
**As we've alluded to a little bit throughout our conversation, there are a lot of people in the Kubernetes community who want a better term, and had "a lot of experience" at running these systems. Then there are, of course, a lot of people who are just at the beginning of their careers; like yourself, at university. How do you see the difference between how those groups interact? Is there one team throughout, or what do you think that each can learn from the other?**<br>
```

Reply:
suggestion is not grammatically correct, and isn't reflective of what was spoken